patches/000-initial-setup.patch

```diff
@@ -1132,32 +1129,6 @@ index 314016979a..323d683788 100644
  }
  
  type x25519Parameters struct {
-diff --git a/src/crypto/x509/boring.go b/src/crypto/x509/boring.go
-index 4aae90570d..42706f93c4 100644
---- a/src/crypto/x509/boring.go
-+++ b/src/crypto/x509/boring.go
-@@ -26,7 +26,7 @@ func boringAllowCert(c *Certificate) bool {
- 	default:
- 		return false
- 	case *rsa.PublicKey:
--		if size := k.N.BitLen(); size != 2048 && size != 3072 {
-+		if size := k.N.BitLen(); size != 2048 && size != 3072 && size != 4096 {
```

Comment:
Upstream added support for 4096 bit keys here, so we don't need to carry this change anymore.